scripts/run_benches_for_runtime.sh

```diff
@@ -9,6 +9,7 @@ chain=$([ "$1" == "spiritnet" ] && echo "spiritnet-dev" || echo "dev")
 standard_args="--release --locked --features=runtime-benchmarks --bin=kilt-parachain"
 
 pallets=(
+	pallet-migration
```

Comment:
Same for this, no?

Reply:
`pallet migration` is in Spiritnet and Peregrine present. It will be released in 1.12